README.md

```diff
@@ -82,6 +82,10 @@ $ make docker-build
 #### Install Harbor
 [Harbor](https://github.com/goharbor/harbor) is registry for the training model in klever-model-registry, please refer to the installation of Harbor [harbor-helm installation](https://github.com/goharbor/harbor-helm)
 
+#### Install Seldon Core
+
+Please have a look at [docs/installer/seldon-core/install.md](docs/installer/seldon-core/install.md)
```

Comment:
Should we need a new page of installation or we can just write it in this doc?

Reply:
I have no opinion on it. WDYT @FogDong

Reply:
Comments are addressed, PTAL @FogDong @gaocegege